galaxy.yml

```diff
@@ -24,4 +24,4 @@ dependencies:
   ansible.posix: "*"
   community.general: "*"
   community.mysql: "*"
-  community.postgresql: "*"
+  community.postgresql: ">=2.0.0"
```

Comment:
Is there a reason for this change?  This would be a breaking change.

Reply:
Not really, no. It's just as far back as I tested the collection. I went to galaxy to find the earliest version, 0.1.0. And that seems to work. Will undo this pinning.